x/superfluid/keeper/twap_price_test.go

```diff
@@ -50,3 +50,74 @@ func (suite *KeeperTestSuite) TestOsmoEquivalentMultiplierSetGetDeleteFlow() {
 	multiplier = suite.App.SuperfluidKeeper.GetOsmoEquivalentMultiplier(suite.Ctx, "gamm/pool/1")
 	suite.Require().Equal(multiplier, sdk.NewDec(0))
 }
+
+func (suite *KeeperTestSuite) TestGetSuperfluidOSMOTokens() {
+	suite.SetupTest()
+	minRiskFactor := suite.App.SuperfluidKeeper.GetParams(suite.Ctx).MinimumRiskFactor
+	poolCoins := sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(1000000000000000000)), sdk.NewCoin("foo", sdk.NewInt(1000000000000000000)))
+	suite.PrepareBalancerPoolWithCoins(poolCoins...)
+	suite.PrepareConcentratedPoolWithCoinsAndFullRangePosition("stake", "foo")
+
+	gammShareDenom := "gamm/pool/1"
+	clShareDenom := "cl/pool/2/"
+	clShareDenomFull := "cl/pool/2/1"
```

Comment:
Lol yeah I had to manually change these when I removed position ID. I am going to do that in PR 5 and link it here because we no longer use position ID in the next PR

Reply:
Added here in PR 5 https://github.com/osmosis-labs/osmosis/pull/4885/commits/4634bd74cf0bf6303e59d1e5b2f523bc15da77a2